api/file-translations/params/fts-callback-parameter.ts

```diff
@@ -0,0 +1,14 @@
+type FtsCallbackUserData = Record<string, unknown>;
+
+enum FtsCallbackMethod {
+    // Other HTTP methods are not supported by API for now
+    POST = "POST"
+}
+
+interface FtsCallback {
+    url: string;
+    httpMethod: FtsCallbackMethod;
+    userData?: FtsCallbackUserData | string
+}
```

Comment:
Please extract to separate files

Reply:
done
1. removed FtsCallbackUserData at all
2. extracted enum to separate file. add it to index.
3. changed `userData?: string`